code/modules/power/supermatter/supermatter.dm

```diff
@@ -1,21 +1,33 @@
-//Ported from /vg/station13, which was in turn forked from baystation12;
-//Please do not bother them with bugs from this port, however, as it has been modified quite a bit.
-//Modifications include removing the world-ending full supermatter variation, and leaving only the shard.
-
-#define NITROGEN_RETARDATION_FACTOR 2        //Higher == N2 slows reaction more
-#define THERMAL_RELEASE_MODIFIER 5                //Higher == less heat released during reaction
-#define PLASMA_RELEASE_MODIFIER 750                //Higher == less plasma released by reaction
-#define OXYGEN_RELEASE_MODIFIER 325        //Higher == less oxygen released at high temperature/power
-#define REACTION_POWER_MODIFIER 0.55                //Higher == more overall power
-
-
-//These would be what you would get at point blank, decreases with distance
+#define NITROGEN_RETARDATION_FACTOR 0.15	//Higher == N2 slows reaction more
```

Comment:
Hmm; this is supposed to be a TG port, yet this is vastly different from what's on TG. Did you do another port? (It's possible they just updated in the mean time), or did you do an older version?

Reply:
It's more Bay then TG, with our older Atmos system we are unable to Port freon.  Updating ours was more straightforward then doing a full port of TG then rebalancing to our atmospherics.

Reply:
Looked through TG's most recent implementation; it definitely doesn't utilize freon; it interacts with plasma, carbon-dioxide, oxygen, and nitrous oxide, but definitely not freon; we have all the gasses it uses.

Also uh, apparently freon is dead on TG; they have a bunch of other gases now (none of which are used by the supermatter).